src/System.IO.Pipelines/ref/System.IO.Pipelines.csproj

```diff
@@ -6,18 +6,22 @@
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netstandard-Debug|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netstandard-Release|AnyCPU'" />
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netstandard1.1-Debug|AnyCPU'" />
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netstandard1.1-Release|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netstandard1.3-Debug|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netstandard1.3-Release|AnyCPU'" />
   <ItemGroup>
     <Compile Include="System.IO.Pipelines.cs" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetGroup)' == 'netstandard1.1'">
+  <ItemGroup Condition="'$(TargetGroup)' == 'netstandard'">
```

Comment:
I don't think this is necessary.

Reply:
https://github.com/dotnet/corefx/blob/6c5a24394309297e03581cd72daefe83ea5f0903/src/System.Threading.Channels/ref/System.Threading.Channels.csproj#L12-L18 ??

Reply:
```csproj
<ItemGroup Condition="'$(TargetGroup)' == 'netstandard'"> 
   <Reference Include="netstandard" /> 
 </ItemGroup> 
```

For TargetGroup = netstandard, I think the reference to netstandard is auto-magically included and we don't need to explicitly  add it.

Reply:
> corefx/src/System.Threading.Channels/ref/System.Threading.Channels.csproj

This is surprising to me.

Reply:
Yea, I can't figure out the build system. Usually in situations like these, copying from another project that does similar things is how I'd work around it 😄.